scripts/travis/imap-stop.sh

```diff
@@ -0,0 +1,2 @@
+dovecot -c /home/stephen/freeradius-server/build/ci/dovecot/etc/dovecot.conf stop
```

Comment:
`$(BUILDIR)` instead of `/home/stephen` ?

Reply:
`imap-stop.sh` is automatically generated, those paths are echoed in from imap-setup.sh. It's no longer included in the commit, because it will be generated when needed.